### PR TITLE
Update French translations

### PR DESCRIPTION
**Pull request by yoan8306**

### CODE REVIEW

**Comment by waydabber** [comment]
Hi @the0neyouseek - can you have a look at this one? Yesterday there is a different one and since I don't speak French I can't decide on this. Thank you! :)

File: MonitorControl/Support/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -62,13 +62,13 @@
...
 
 /* Shown in the alert dialog */
-"Reset Preferences?" = "Réinitialiser les préférences ?";
+"Reset Preferences?" = "Réinitialiser les réglages ?";
 
 /* Shown in the alert dialog */
 "Safe Mode Activated" = "Mode sans échec activé";
 
 /* Shown in the alert dialog */
-"Shift was pressed during launch. MonitorControl started in safe mode. Default preferences are reloaded, DDC read is blocked." = "Maj. appuyée pendant le démarrage. MonitorControl démarré en mode sans échec. Les préférences par défaut sont utilisées, la communication via DDC est bloquée.";
```
Comment (the0neyouseek) [inline MonitorControl/Support/fr.lproj/Localizable.strings:71]: Cette traduction est plus logique. L'application a déjà démarré

File: MonitorControl/UI/fr.lproj/Main.strings [hunk 2]
```diff
@@ -2,82 +2,82 @@
...
-"95V-M4-2l5.title" = "Special thanks to our contributors!";
+"95V-M4-2l5.title" = "Remerciements spécial à nos contributeurs!";
 
 /* Class = "NSButtonCell"; title = "Hide Menu Icon"; ObjectID = "aXL-i8-S6R"; */
-"aXL-i8-S6R.title" = "Hide Menu Icon";
+"aXL-i8-S6R.title" = "Cacher l'icône dans le menu";
 
 /* Class = "NSButtonCell"; title = "Disable macOS volume OSD"; ObjectID = "bkM-Px-U3b"; */
-"bkM-Px-U3b.title" = "Disable macOS volume OSD";
+"bkM-Px-U3b.title" = "Désactive le volume OSD";
```
Comment (the0neyouseek) [inline MonitorControl/UI/fr.lproj/Main.strings:11]: Plutôt "Désactive l'OSD de volume macOS"

File: MonitorControl/UI/fr.lproj/Main.strings [hunk 3]
```diff
@@ -2,82 +2,82 @@
...
 
 /* Class = "NSButtonCell"; title = "Reset settings"; ObjectID = "BYS-7Y-bRz"; */
-"BYS-7Y-bRz.title" = "Reset settings";
+"BYS-7Y-bRz.title" = "Réinitialiser les réglages";
 
 /* Class = "NSButtonCell"; title = "Enable Mute DDC command"; ObjectID = "bZq-0d-lJa"; */
-"bZq-0d-lJa.title" = "Enable Mute DDC command";
+"bZq-0d-lJa.title" = "Active Mute commande DDC";
```
Comment (the0neyouseek) [inline MonitorControl/UI/fr.lproj/Main.strings:17]: Active la commande DDC "Muet" (ou "Sourdine" ?)

File: MonitorControl/UI/fr.lproj/Main.strings [hunk 4]
```diff
@@ -2,82 +2,82 @@
...
 
 /* Class = "NSTextFieldCell"; title = "Keyboard:"; ObjectID = "c7N-cO-759"; */
-"c7N-cO-759.title" = "Keyboard:";
+"c7N-cO-759.title" = "Clavier:";
 
 /* Class = "NSMenuItem"; title = "Custom"; ObjectID = "Cle-DD-vR7"; */
-"Cle-DD-vR7.title" = "Custom";
+"Cle-DD-vR7.title" = "Personnaliser";
```
Comment (the0neyouseek) [inline MonitorControl/UI/fr.lproj/Main.strings:23]: Personnalisé

**Review by the0neyouseek — CHANGES_REQUESTED** [review]
I've left some comments (in French) if you can, have a look. Otherwise, everything looks good to me

**Comment by yoan8306** [comment]
Hi, 
my answer in  green below.
Thanks :)
Regards

File: MonitorControl/UI/fr.lproj/Main.strings (hunk 3 above)
Comment (yoan8306) [reply MonitorControl/UI/fr.lproj/Main.strings:17]: "Active la commande DDC sourdine"

File: MonitorControl/Support/fr.lproj/Localizable.strings (hunk 1 above)
Comment (yoan8306) [reply MonitorControl/Support/fr.lproj/Localizable.strings:71]: Maintenir Maj. appuyé pendant le démarrage.

**Pull request closed without merging**